misc/docs/source/tutorial/code_snippets/trigger_tutorial.rst

```diff
@@ -396,6 +397,27 @@ For :func:`~obspy.signal.trigger.ar_pick`, input and output are in seconds.
 This gives the output 30.6350002289 and 31.2800006866, meaning that a P pick at
 30.64s and an S pick at 31.28s were identified.
 
+AIC - Akaike Information Criterion
+===========
+
+For :func:`~obspy.signal.trigger.aic`, the input is a numpy.ndarray and the
+output is in sample.
+
+    >>> from obspy.core import read, UTCDateTime
+    >>> from obspy.signal.trigger import aic
+    >>> trace = read("https://examples.obspy.org/ev0_6.a01.gse2")[0]
+    >>> df = trace.stats.sampling_rate
+    >>> trace_s = trace.slice(UTCDateTime('1970-01-01T01:00:31.6'),
+                              UTCDateTime('1970-01-01T01:00:34.3'))
```

Comment:
This should be:

```
    ...                        UTCDateTime('1970-01-01T01:00:34.3'))
```
i.e. 3 dots where the Python prompt arrows are in the first line

Reply:
updated, I was missing this